d2l-hm-constants-behavior.html

```diff
@@ -20,7 +20,8 @@
 			root: 'https://api.brightspace.com/rels/root',
 			thumbnailRegular: 'https://api.brightspace.com/rels/thumbnail#regular',
 			thumbnailSmall: 'https://api.brightspace.com/rels/thumbnail#small',
-			userProfile: 'https://api.brightspace.com/rels/user-profile'
```

Comment:
Huh, strange. I would have expected this to be `https://users.api.brightspace.com/rels/profile` instead - then again, that would make the one you added rather hard to define... `https://users.api.brightspace.com/rels/<undefined>`? :P